include/ck/tensor_operation/gpu/element/element_wise_operation.hpp

```diff
@@ -528,6 +528,26 @@ struct UnaryTypeConvert<ck::bhalf_t, float>
     }
 };
 
+struct ConvScale
```

Comment:
You can add next operator in  include/ck/tensor_operation/gpu/element/combined_element_wise_operation.hpp?
for example `QuadrupleWithUnaryCombinedOp` and use `QuadrupleWithUnaryCombinedOp<Multiply, Multiply,Multiply, PassThrough, PassThrough,PassThrough>`

But it looks a little bit complicated so feel free to skip this comment and add new Elementwise Op :)

Reply:
Wow, meta-elementwise-op, I haven't thought about it! Let's keep it simple so far and once we add all required elementwise ops, we can generalize with a meta-op, what do you think?

Reply:
Sure!